module-audio/Audio/Profiles/README.md

```diff
@@ -0,0 +1,50 @@
+# Profile JSON file
+This document aims to describe the profile file and how to tweak those values.
+
+## First things first
+The JSON files describing the profile parameters are used to configure the audio profile with proper input/output and filter settings. 
+It is **NOT** recommended to change anything than the filter settings as it might make your device speechless.
```

Comment:
Good idea to point it out! :+1:

Reply:
Would it be possible to add a fallback configuration or it is tedious to validate provided configs?

Reply:
Yep, I'm actually rewriting it a little bit to provide a fallback when file loading/parsing give an error

Reply:
Although it won't work if someone e.g. disables the path in properly formatted JSON file - IMO we shouldn't validate the file in this way as this is not a normal file for user to tweak - if someone does something with this - one should know it's consequences :P